tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_test.ts

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 import {Component, Input, NO_ERRORS_SCHEMA, ViewChild} from '@angular/core';
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 import {By} from '@angular/platform-browser';
-import {KeyType, sendKey, sendKeys} from '../../testing/dom';
+import {sendKeys} from '../../testing/dom';
 import {LinkedTimeFobComponent} from './linked_time_fob_component';
 import {AxisDirection} from './linked_time_fob_controller_component';
 
```

Comment:
In this file you should also be able to use debugElement.query instead of debugElement.nativeElement.querySelector.

Reply:
Changed.